### ISSUE TITLE
WPS317: handle tuples, lists,... in parameters

### ISSUE THREAD

**Issue by slapshin** [wemake-services/wemake-python-styleguide#867]
Hello! I have a suggestion for this violation. For example, we have the code:
```python
demo_func(
    (1, 2),
    (1, 2),
    (1, 3),
)
```

And we have violation WPS317. I understand that "top-level" arguments should be placed vertically, but each argument is a tuple and it will be more beautiful to place members horizontally. Maybe this rule for each nested argument should be processed separately?

**Comment by sobolevn** [comment]
This is a well known bug, I am still not able to fix it. See #https://github.com/wemake-services/wemake-python-styleguide/issues/454

**Comment by sobolevn** [comment]
I am going to provide a different logic, but I am not sure about the time frame.
